.github/CODEOWNERS

```
@@ -1334,7 +1334,27 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 ### END Observability Plugins
 
 # Presentation
-/x-pack/test/disable_ems @elastic/kibana-presentation
+/test/interpreter_functional/baseline @elastic/kibana-presentation
+/test/interpreter_functional/snapshots @elastic/kibana-presentation # Assigned per https://github.com/elastic/kibana/pull/54342
+/test/functional/services/inspector.ts @elastic/kibana-presentation
+/x-pack/test/functional/services/canvas_element.ts @elastic/kibana-presentation
+/x-pack/test/functional/page_objects/canvas_page.ts @elastic/kibana-presentation
+/x-pack/test/accessibility/apps/group3/canvas.ts @elastic/kibana-presentation
+/x-pack/test/upgrade/apps/canvas @elastic/kibana-presentation
+/x-pack/test/upgrade/apps/dashboard @elastic/kibana-presentation
+/test/functional/screenshots/baseline/tsvb_dashboard.png @elastic/kibana-presentation
+/test/functional/screenshots/baseline/dashboard_*.png @elastic/kibana-presentation
+/test/functional/screenshots/baseline/area_chart.png @elastic/kibana-presentation
+/test/functional/page_objects/share_page.ts @elastic/kibana-presentation
```

```suggestion
/test/functional/page_objects/share_page.ts @elastic/appex-sharedux
```

It seems like this might be more applicable to @elastic/appex-sharedux as they [currently own the share plugin](https://github.com/elastic/kibana/blob/main/src/plugins/share/kibana.jsonc#L5).

---

https://github.com/elastic/kibana/pull/200209/commits/2c763f3511a064c0a5404f29c6b8a56898bbc34d